### DISCUSSION TITLE
Visualizing battery automations

### DISCUSSION THREAD

**Discussion by madpilot** [amberelectric/public-api#112] [Q&A]
Hey gang,

We are about to start iterating on the Smartshift battery UI, and we were wondering if we could talk to you.

We are particularly interested in people with batteries and EVs (we don't care what brand) who are doing _some_ sort of optimisation on them. You don't even have to be a Smartshift user.

If you are willing to help out, Add a thumbs up and I can contact you privately.

**Comment by purcell-lab** [comment]
Getting good results with EMHASS. 

![Screenshot 2022-06-15 21 53 22](https://user-images.githubusercontent.com/79175134/173820984-d7536331-6f8a-42e3-8ca6-47a3e6fed344.png)

**Comment by purcell-lab** [comment]
My goto home page for HomeAssistant provides me Amber Price gauge,  real time power flow (kW) and energy (kWh) as well as controls for my battery.  The default HomeAssistant Energy dashboard doesn't show realtime power flow.



https://user-images.githubusercontent.com/79175134/173820663-7bbe54ed-7bf5-4ebf-a389-342f60087ed9.mp4
